core/templates/pages/contributor-dashboard-page/translation-language-selector/translation-language-selector.component.ts

```diff
@@ -102,6 +105,16 @@ export class TranslationLanguageSelectorComponent implements OnInit {
 
   toggleDropdown(): void {
     this.dropdownShown = !this.dropdownShown;
+    if (this.dropdownShown) {
+      this.optionsFilter = '';
+      this.filteredOptions = this.options;
+      setTimeout(() => {
+        if (this.filterDivRef) {
+          this.filterDivRef.nativeElement.focus();
+        }
+      }
+      , 1);
+    }
```

Comment:
Can you update/add the frontend tests for `toggleDropdown` function?

Reply:
Added, thanks!